spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -1179,7 +1179,12 @@ def linenumberarea_paint_event(self, event):
         active_line_number = active_block.blockNumber() + 1
 
         def draw_pixmap(ytop, pixmap):
-            painter.drawPixmap(0, ytop + (font_height-pixmap.height()) / 2,
+            if is_pyqt46:

```

Comment:
Nop, it needs to be more fine-grained because we support PyQt >=5.2. So the solution is something along the lines

```python
from qtpy import QT_VERSION
QT55_VERSION = programs.check_version(QT_VERSION, "5.5", ">=")

if not QT55_VERSION:
    pixmap_height = pixmap.height()
else:
    ....
```

Of course, you need to add the imports and constants in the right places.